From 335d68f451d351d3d4cad7be2cdc07463e8aea8c Mon Sep 17 00:00:00 2001
From: Nils Philippsen <nils@redhat.com>
Date: Tue, 10 Jun 2025 16:22:04 +0200
Subject: [PATCH] Remove shebang from bash autocompletion
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The autocompletion snippet has to be sourced from a shell, but running
it in its own process doesn’t make sense.

Signed-off-by: Nils Philippsen <nils@redhat.com>
---
 contrib/autocompletion/bash_autocomplete | 1 -
 1 file changed, 1 deletion(-)

diff --git a/contrib/autocompletion/bash_autocomplete b/contrib/autocompletion/bash_autocomplete
index 5cb62f26a7..97c526e9a7 100755
--- a/contrib/autocompletion/bash_autocomplete
+++ b/contrib/autocompletion/bash_autocomplete
@@ -1,4 +1,3 @@
-#! /bin/bash
 # Heavily inspired by https://github.com/urfave/cli
 
 _cli_bash_autocomplete() {
-- 
2.45.2

